.github/workflows/version.yml

```diff
@@ -33,10 +33,22 @@ jobs:
               run: git checkout -b version-bump-${{ github.sha }}
 
             - name: Generate version
+              id: bumpVersion
               uses: Expensify/Expensify.cash/.github/actions/bumpVersion@master
               with:
                   GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
 
+            - name: Commit and tag new version
+              run: |
+                git add \
+                  ./package.json \
+                  ./package-lock.json \
+                  ./android/app/build.gradle \
+                  ./ios/ExpensifyCash/Info.plist \
+                  ./ios/ExpensifyCashTests/Info.plist
+                git cm -m "Update version to ${{ steps.bumpVersion.outputs.newVersion }}"
```

Comment:
Is `cm` a local shortcut you have installed locally?

Reply:
😅 is `git commit` even a command?

Reply:
Good catch! I honestly forgot that `git cm` is just an alias